lib/usd/translators/meshWriter_BlendShapes.cpp

```diff
@@ -315,14 +311,12 @@ MStatus mayaGetBlendShapeInfosForMesh(
         // NOTE: (yliangsiew) Because this can end up grabbing the wrong plug (i.e.
         // blendShape.weight[1]), we double-check here and advance the iterator if necessary.
         if (!outputGeomElemPlug.isElement()) {
-            itDg.next();
             continue;
         }
         MPlug outputGeomPlug = outputGeomElemPlug.array(&stat);
         CHECK_MSTATUS_AND_RETURN_IT(stat);
         MString outputGeomPlugName = outputGeomPlug.partialName(0, 0, 0, 0, 0, 1);
         if (outputGeomPlugName != "outputGeometry") {
-            itDg.next();
```

Comment:
This is inside a for loop which already calls .next().
My understand is that there might be a hidden bug here where we can accidentally skip some elements.